### ISSUE TITLE
Update link in github config

### ISSUE THREAD

**Issue by madiedgar** [legesher/legesher-translations#123]
**Update Github Config Link**
The link in the `.github/config.yml` is incomplete. Please update the link to include the `.html` extension at the end of the url. 

It should look like this:

``` yml
We'd love to celebrate you and your contribution to the community. Let us say "Thank You" again, by sending you a [special gift](https://legesher.io/dreamteam.html) only given to special individuals like you who have chosen to give back.
```

**Comment by arjunshekhawat5** [comment]
Hello, Can I be assigned to this issue. I am new to open source and would love to make my first open source contribution to this project.
Thank you

**Comment by madiedgar** [comment]
Hey @arjunshekhawat5 👋 That would be amazing!! I'll assign you to it! If you need any help, feel free to reach out 😄

**Comment by madiedgar** [comment]
Closing this as it was resolved by @arjunshekhawat5 in #124
